css/src/typography.js

```diff
@@ -0,0 +1,27 @@
+import {css, injectGlobal} from 'styled-components';
+import { font } from '@nulogy/tokens';
+
+const fontCSS = {
+  // Size
+  'font-size--smaller': css`font-size: ${font.size.smaller}px`,
+  'font-size--small': css`font-size: ${font.size.small}px`,
+  'font-size--medium': css`font-size: ${font.size.medium}px`,
+  'font-size--large': css`font-size: ${font.size.large}px`,
+  'font-size--larger': css`font-size: ${font.size.larger}px`,
+  'font-size--largest': css`font-size: ${font.size.largest}px`,
+  'font-size--page-title': css`font-size: ${font.size.pageTitle}px`,
+  'font-size--section-title': css`font-size: ${font.size.sectionTitle}px`,
+  'font-size--subsection-title': css`font-size: ${font.size.sectionTitle}px`,
+
+  // Family
+  'font-family--regular': css`font-family: ${font.family.regular}`,
+  'font-family--mono': css`font-family: ${font.family.mono}`,
+
+  // Weight
+  'font-weight--bold': css`font-weight: ${font.weight.bold}`,
+  'line-height--regular': css`line-height: ${font.lineHeight.regular}`,
+}
```

Comment:
Would you be interested in generating this dynamically based on the keys in the tokens?

Something like:

```js
const buildMixins = name => Object.entries(font[name]).reduce((result, [key, value]) => ({ 
  ...result, 
  [`font-${name}--${key}`]: css`font-${name}: ${value}px` 
}), {});

const fontSize = buildMixins('size');
const fontFamily = buildMixins('family');
const fontWeight = buildMixins('weight');

Object.entries({ ...fontSize, ...fontFamily, ...fontWeight })
  .forEach(([className, rule]) => injectGlobal`.${className} { ${rule} }`);
```

I'm curious about your thoughts @matthewlyle @nikola-nulogy about the tursness of something like this.

One advantage of an approach like this is that if the keys were to change in the tokens, the css would update automatically.

On the other hand its much harder to read.

Reply:
I would prefer if we kept the original version.  It is more readable and familiar to a designer. This would make it less fragile in case designer is making the edits.